lib/core/presentation/bloc/bloc_observer.dart

```diff
@@ -0,0 +1,28 @@
+// ignore_for_file: avoid_print
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class MyBlocObserver extends BlocObserver {
```

Comment:
Are you interested in logging the events and states? Notice that you are using print statements too, If this is for debuging, Try using `debugPrint`.